### PR TITLE
Speed up EPG loading

### PR DESCRIPTION
**Pull request by vpeter4**
With #269 there are two regex which are evaluated on every function call. In my case loading time was increased from 800msec to 10sec. This PR replaces those regex with static ones.
For Leia branch.

### CODE REVIEW

**Comment by phunkyfish** [comment]
Thanks for this vpeter. Can you please create the PR for Matrix first?

Once that's merged the Leia version can go in.

BTW, you also need to modify the changelog and addon.xml.in. Just increment the patch number as this is a fix.

**Comment by vpeter4** [comment]
Done.

**Comment by vpeter4** [comment]
Aligned with Matrix.

Now loading is so fast that I don't even see notification anymore :-)

**Review by phunkyfish — APPROVED** [review]
Looks good. Will merge once CI is happy.

**Pull request merged**
